sphinx_doc/getting_started/index.rst

```diff
@@ -54,7 +54,7 @@ Running the Example
 
    .. code-block:: bash
 
-      ros2 launch nav2_bringup nav2_tb3_simulation_launch.py
+      ros2 launch nav2_bringup nav2_simulation_launch.py
```

Comment:
It is `nav2_tb3_simulation_launch`, there's no `nav2_simulation_launch` file

Reply:
If you look at the directions above this line, it says to install dashing from binaries and `source /opt/ros/dashing/setup.bash` so these instructions are currently for dashing.

We released for eloquent, but we missed the eloquent sync, so binaries aren't available yet. If we want people to move to eloquent, we need to update this to eloquent.

For now I was just trying to make it correct for dashing. As it is, it's wrong for both.

Reply:
Fair enough. I didnt notice the dashing reference above. 

I’d personally like to future proof this with <rosdistro> and remove ALL references to a specific distribution. Its hard enough to maintain docs, let alone N versions of it. 

But I suppose there’s nothing wrong with this here then.

Reply:
This probably deserves a separate conversation around support for dashing vs. eloquent, since dashing is an LTS and eloquent isn't. 

I agree in general using <rosdistro> would be ideal, but in this case the filename is different depending on which one is used. So we may have to maintain at least 2 options for the documentation here for now.

Reply:
Oops that's supposed to say in general using `rosdistro` is ideal